Airports/EGBB/Sids.txt

```diff
@@ -10,12 +10,3 @@ SID:EGBB:15:TNT4G:D144B TNT33 TNT30 D177R TNT
 SID:EGBB:33:UMLUX1M:BBW01 BBW02 BBW04 UMLUX
 SID:EGBB:33:UNGAP1M:BBN31 BBE21 BBE20 UNGAP
 SID:EGBB:15:WCO2Y:BBS02 BBS04 BBS06 BBN13 BBN17 BBE22 WCO
-
-;withdrawn 2019/06
-SID:EGBB:33:#ADMEX1D:D330B D317Z D317T CPT55 ADMEX
-SID:EGBB:33:#TNT4D:D330B D189W D189S TNT
-SID:EGBB:15:#TNT6E:D144B TNT33 DTY32 D184Z TNT
-SID:EGBB:33:#UNGAP1D:D330B D317Z D317R UNGAP
-SID:EGBB:33:#WCO5D:D330B D317Z D317T DTY18 D313P WCO
-SID:EGBB:15:#WHI1L:BBE05 BBE07 BBN11 BBN18 WHI
-SID:EGBB:33:#WHI4D:D330B D327T D327X HON27 WHI
```

Comment:
(Some) of these fixes can also be removed as they're no longer used - can you check and remove those which have no further purpose?